src/App/reducers/index.js

```diff
@@ -1,9 +0,0 @@
-import {combineReducers} from 'redux';
```

Comment:
This state **belongs to** the application, it isn't a sibling of the application. Moving it up a level into a `state` folder is confusing for me. Can you explain the advantages?